<launch>
    <node name="tplsearch_test" pkg="tplsearch_test" type="tplsearch_test_node" output="screen">
        <param name="autostart" value="0"/>
        <param name="runs" value="3"/>

        <param name="object" value="Cylinder"/>
        <param name="model_parameter_1" value="0.05"/>
        <param name="model_parameter_2" value="0.01"/>
        <param name="model_parameter_3" value="200"/>

        <param name="animation" value="TestAdvanceFaceTracker"/>
        <!--  RotationPlaneZ RotationCylinder2 TranslationCylinder1 TranslationPlane1 -->

        <param name="list_length" value="10"/>
        <param name="template_keywords" value="santa christmas"/>
        <param name="background_keywords" value="city landscape forest"/>
        <param name="render_image_width" value="500"/>
        <param name="render_image_height" value="500"/>

        <param name="fps" value="1"/>
        <param name="frames" value="30"/>

        <param name="skip_image_download" value="true"/>
        <param name="skip_ground_truth" value="true"/>
        <param name="skip_render" value="false"/>
        <param name="skip_img_list" value="true"/>
    </node>
</launch>
